grafana/overlays/emea/devconfus2021/opf-monitoring/routes/route.yaml

```diff
@@ -0,0 +1,15 @@
+---
+kind: Route
+apiVersion: route.openshift.io/v1
+metadata:
+  name: grafana
+spec:
+  host: grafana-route-opf-monitoring.apps.devconfus2021.aws.operate-first.cloud
```

Comment:
Route doesn't need a host, if not set it's inferred automatically.

Reply:
Resolved in latest commit.